web/template/pa-hub/pa-deputy-details.gotmpl

```diff
@@ -12,6 +12,14 @@
         </h1>
         <div class="moj-button-menu">
             <div class="moj-button-menu__wrapper">
+                <a
+                    href="{{ prefix (printf "/%d/delete-deputy" .DeputyDetails.ID) }}"
+                    role="button"
+                    draggable="false"
+                    class="govuk-button moj-button-menu__item govuk-button--secondary {{if eq .UserDetails.IsSystemManager false}}{{if eq .DeputyDetails.CanDelete true}} hide{{end}}{{end}}"
```

Comment:
`{{if eq .UserDetails.IsSystemManager false}}` can be `{{if not .UserDetails.IsSystemManager}}`.

You could also combine them both with `{{if and (not .UserDetails.IsSystemManager) .DeputyDetails.CanDelete }}`

Reply:
done